### ISSUE TITLE
User cannot see the usernames of other users within a `chat_group`

### ISSUE THREAD

**Issue by GabrielInTheWorld** [OpenSlides/openslides-autoupdate-service#373]
I sent this request:
```js
{
    "collection": "chat_group",
    "ids": [
        4
    ],
    "fields": {
        "chat_message_ids": {
            "type": "relation-list",
            "collection": "chat_message",
            "fields": {
                "chat_group_id": null,
                "content": null,
                "created": null,
                "user_id": {
                    "type": "relation",
                    "collection": "user",
                    "fields": {
                        "title": null,
                        "first_name": null,
                        "last_name": null,
                        "username": null,
                        "id": null
                    }
                },
                "id": null
            }
        },
        "name": null,
        "weight": null,
        "read_group_ids": {
            "type": "relation-list",
            "collection": "group",
            "fields": {
                "name": null,
                "permissions": null,
                "admin_group_for_meeting_id": null,
                "default_group_for_meeting_id": null,
                "id": null
            }
        },
        "write_group_ids": {
            "type": "relation-list",
            "collection": "group",
            "fields": {
                "name": null,
                "permissions": null,
                "admin_group_for_meeting_id": null,
                "default_group_for_meeting_id": null,
                "id": null
            }
        },
        "id": null
    }
}
```

and got this response:
```js
{
    "chat_group/4/chat_message_ids": [
        46
    ],
    "chat_group/4/id": 4,
    "chat_group/4/name": "www",
    "chat_group/4/read_group_ids": [
        7,
        8
    ],
    "chat_group/4/weight": 2,
    "chat_group/4/write_group_ids": [
        8
    ],
    "chat_message/46/chat_group_id": 4,
    "chat_message/46/content": "haha",
    "chat_message/46/created": 1637835437,
    "chat_message/46/id": 46,
    "group/7/admin_group_for_meeting_id": 2,
    "group/7/id": 7,
    "group/7/name": "Admin",
    "group/7/permissions": [],
    "group/8/id": 8,
    "group/8/name": "Delegates",
    "group/8/permissions": [
        "agenda_item.can_see_internal",
        "assignment.can_nominate_other",
        "assignment.can_nominate_self",
        "list_of_speakers.can_be_speaker",
        "mediafile.can_see",
        "meeting.can_see_autopilot",
        "meeting.can_see_frontpage",
        "motion.can_create",
        "motion.can_create_amendments",
        "motion.can_support",
        "projector.can_see",
        "user.can_see"
    ]
}
```

As you can see, the user of the `chat_message/46` is missing in the response.

**Comment by ostcar** [comment]
I guess you don't send the request with a superadmin but with a different user. Which permissions does your request user have? Is your request user allowed to see users?

Maybe the bug is, that the user-restriction has to be adjusted: https://github.com/OpenSlides/OpenSlides/wiki/user-restrictions

There is the part in the __see__ property:
```
There is a related object:

    There exists a motion which Y can see and X is a submitter/supporter.
    There exists an option which Y can see and X is the linked content object.
    There exists an assignment candidate which Y can see and X is the linked user.
    There exists a speaker which Y can see and X is the linked user.
    There exists a poll where Y can see the poll/voted_ids and X is part of that list.
    There exists a vote which Y can see and X is linked in user_id or delegated_user_id.
```

Maybe there has to be a new condition:
```
    There exists a chat_message which Y can see and X is the linked in chat_message/user_id
```

**Comment by GabrielInTheWorld** [comment]
> I guess you don't send the request with a superadmin but with a different user. Which permissions does your request user have? Is your request user allowed to see users?
> 
> Maybe the bug is, that the user-restriction has to be adjusted: https://github.com/OpenSlides/OpenSlides/wiki/user-restrictions
> 
> There is the part in the **see** property:
> 
> ```
> There is a related object:
> 
>     There exists a motion which Y can see and X is a submitter/supporter.
>     There exists an option which Y can see and X is the linked content object.
>     There exists an assignment candidate which Y can see and X is the linked user.
>     There exists a speaker which Y can see and X is the linked user.
>     There exists a poll where Y can see the poll/voted_ids and X is part of that list.
>     There exists a vote which Y can see and X is linked in user_id or delegated_user_id.
> ```
> 
> Maybe there has to be a new condition:
> 
> ```
>     There exists a chat_message which Y can see and X is the linked in chat_message/user_id
> ```

Yes, we can adjust those restrictions. But, yes, I sent this request as a superadmin.

**Comment by GabrielInTheWorld** [comment]
I updated the [user_restrictions](https://github.com/OpenSlides/OpenSlides/wiki/user-restrictions).

**Comment by ostcar** [comment]
> I updated the [user_restrictions](https://github.com/OpenSlides/OpenSlides/wiki/user-restrictions).

I added a commit in #371 



> But, yes, I sent this request as a superadmin.

This is strange. I would like to reproduce this. Can you tell me (step by step) what I have to do? I can not find the chat in the UI.

**Comment by GabrielInTheWorld** [comment]
> > But, yes, I sent this request as a superadmin.
> 
> This is strange. I would like to reproduce this. Can you tell me (step by step) what I have to do? I can not find the chat in the UI.

Well, you cannot, because the chat is "work in progress", at the moment :D

**Comment by ostcar** [comment]
This should be fixed or be a duplicate of #379. If not, then please reopen.
